tensorflow/core/kernels/mkl_pooling_ops_common.cc

```diff
@@ -106,20 +118,30 @@ void MklPoolingFwdPrimitive<T>::Execute(const T* src_data, T* dst_data,
   context_.src_mem->set_data_handle(
       static_cast<void*>(const_cast<T*>(src_data)));
   context_.dst_mem->set_data_handle(static_cast<void*>(dst_data));
-  if (context_.alg_kind == pooling_max &&
+  if (context_.alg_kind == ALGORITHM::pooling_max &&
       context_.prop_kind ==
-          prop_kind::forward_training) {  // max pooling must have ws
+          prop_kind::forward_training) {  // Max pooling must have workspace.
     DCHECK(ws_data != nullptr);
     context_.ws_mem->set_data_handle(ws_data);
   }
+
+#ifdef ENABLE_MKLDNN_V1
+  //ExecuteFwdPrimitives(context_);
```

Comment:
Please delete this line.

Reply:
Done! Thank you for catching it!

Reply:
The suggested code changes seems reasonable and minimal changes I have changed with suggested code change. Looks clean and thank you for the suggestion.